<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Results</class>
 <widget class="QDialog" name="Results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Результаты тестирования!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(36, 113, 190, 255), stop:1 rgba(243, 243, 243, 255));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="Button_PDF">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
	color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Выгрузить в PDF</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="Button_Send">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>331</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>331</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Отправить преподавателю</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>415</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
     <property name="text">
      <string>Ваше имя:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit_Name">
     <property name="minimumSize">
      <size>
       <width>831</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border-radius: 7px;
border: 3px solid rgb(0, 135, 0); 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 180, 34, 255), stop:1 rgba(152, 255, 152, 255));
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QListWidget" name="listWidget_Result">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>982</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
border-radius: 0px;
border: 3px solid rgb(31, 99, 167);  
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(171, 181, 190, 255), stop:1 rgba(243, 243, 243, 255));
}

QListWidget::item:selected {
border-radius: 0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0.551, x2:1, y2:0.557, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(83, 255, 83, 255));
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>822</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="Button_Close">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 180, 0, 255), stop:1 rgba(152, 255, 152, 255));
}

QPushButton:hover {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 	
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(36, 232, 36, 255), stop:1 rgba(236, 250, 236, 255));
}

QPushButton:pressed {
border-radius: 20px;
border: 3px solid rgb(0, 135, 0); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(236, 250, 236, 255), stop:1 rgba(36, 232, 36, 255));
}

QPushButton:disabled {
border-radius: 20px;
border: 3px solid rgb(90, 90, 90); 
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(210, 210, 210, 255));
}</string>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
